README.md

```diff
@@ -13,3 +14,36 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+1. Asymptotic analysis observes the algorithm without considering constant
+   factors, so while they impact the runtime, they don't show up in the
+   asyptotic analysis.
+2. While asymptotic analysis can tell us about the theoretical runtime of a program
+   for very large input sizes, it doesn't account for practical uses of the algorithm,
+   so other factors like memory usage might throttle the program if the algorithm was
+   implemented poorly.
```

Comment:
We can analyze memory complexity, and even if an implementation is not good we can still analyze it.